docs/usage-notes.md

```diff
@@ -22,6 +23,29 @@ Once the app is launched (either web or desktop version), it will prompt you to
 
 ![session_select](https://imgur.com/nBOy7zH.png)
 
+## Format of `profiles.json` 
+
+You can choose to load your custom sessions by clicking on the **file icon** beside the session dropdown. 
+
+Following which, submit a json file with the name `profiles.json`, where the format is specified below.
+
+```json
+{
+    "profiles": [
+        {
+            "profileName": "CATcher", 
+            "encodedText": "CATcher-org/public_data"
+        }
+    ]
+}
+```
+
+The json supplied should only consist of **one key-pair value**, where the key is `"profiles"` and the value is an array of `Profiles` supplied, where each `Profile` is an object containing the `profileName` and `encodedText` fields. 
+
+`profileName` refers to the profile name displayed in the session select page. `encodedText` refers to the repository which stores the required settings for your custom session. 
+
+> **Note**: You **must** have both of these fields in each `Profile` and no extra unnecessary fields! Else, the `profile.json` file that you have supplied will not be parsed successfully. 
```

Comment:
I'm currently not checking for unnecessary fields, should we add that functionality in?

Reply:
Ah alright, I think we can leave it out for now! 

When we actually check for unnecessary fields, then one of us can update the dev notes whenever necessary!